code/handlers/trackservers.q

```diff
@@ -235,6 +235,8 @@ addprocs:{[connectiontab;procs;connect]
 
 // addprocscustom is to allow bespoke extensions when adding processes
 addprocscustom:{[connectiontab;procs]}
```

Comment:
If you want an error trap to return a function then you have to wrap the function within the error trap function e.g. 

@[value;.servers.addprocscustom;{{[connectiontab;procs]}}]